Building master branch fails

When running `npm run deploy`, both local and on Netlify, I get this error (Uganda country profile JSON missing):

```
info => Building preview..
info => Loading preview config..
info => Loading presets
info => Using default webpack setup.
info => Loading create-react-app config.
info => Compiling preview..
 70% building 73/80 modules 7 active ...ite-react/CalciteThemeProvider/index.jsERR! => Failed to build the preview
ERR! ./src/stories/index.js
ERR! Cannot find file '../json/countryProfiles/country profile 800 Uganda.json' in './src/stories'.
(node:8546) UnhandledPromiseRejectionWarning: ModuleNotFoundError: Module not found: Error: Can't resolve '../json/countryProfiles/country profile 800 Uganda.json' in '/home/google/Projects/unstats/CountryProfiles/src/stories'
    at factory.create (/home/google/Projects/unstats/CountryProfiles/node_modules/webpack/lib/Compilation.js:823:10)
    at factory (/home/google/Projects/unstats/CountryProfiles/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/google/Projects/unstats/CountryProfiles/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/google/Projects/unstats/CountryProfiles/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/google/Projects/unstats/CountryProfiles/node_modules/neo-async/async.js:2825:7
    at /home/google/Projects/unstats/CountryProfiles/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/home/google/Projects/unstats/CountryProfiles/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
```

--- a/src/stories/NavStory.js
+++ b/src/stories/NavStory.js
@@ -28,24 +28,10 @@ const StyledSection = styled.div`
   padding: 2rem 0;
 `;
 
-const NavStory = props => {
+const NavStory = ({ goals, goalInfos }) => {
   return (
     <StyledContentContainer>
-      <Nav
-        selectedCountry="Italy"
-        goals={[
-          { id: 1, title: 'Goal 1' },
-          { id: 2, title: 'Goal 2' },
-          { id: 3, title: 'Goal 3' },
-          { id: 4, title: 'Goal 4' }
-        ]}
-      />
-      <StyledSectionContainer>
-        <StyledSection id="goal-1-section">Goal 1 Section</StyledSection>
-        <StyledSection id="goal-2-section">Goal 2 Section</StyledSection>
-        <StyledSection id="goal-3-section">Goal 3 Section</StyledSection>
-        <StyledSection id="goal-4-section">Goal 4 Section</StyledSection>
-      </StyledSectionContainer>
+      <Nav selectedCountry="Italy" goals={goals} goalInfos={goalInfos} />
     </StyledContentContainer>
   );
 };

--- a/src/stories/index.js
+++ b/src/stories/index.js
@@ -23,19 +23,25 @@ import UgandaMap from '../assets/uganda_map.png';
 
 // json
 import countries from '../json/countryList.json';
-import country from '../json/countryProfiles/country profile 800 Uganda.json';
+import country from '../json/countryProfiles/country_profile800.json';
 import sdgsMoreInfo from '../json/sdgs-more-info.json';
 
 const goal = country.goals.find(goal => goal.goalCode === '16');
 const timeSeries = goal.targets[0].indicators[0].facts[0];
 const metricSeries = goal.targets[1].indicators[0].facts[0];
+
+const boolGoal = country.goals.find(goal => goal.goalCode === '17');
+const boolSeries = boolGoal.targets[3].indicators[0].facts[0];
+
 const goalInfo = sdgsMoreInfo.data.find(sdgInfo => sdgInfo.goal === 16);
 
 storiesOf('SiteBreadcrumb', module).add('default', () => <SiteBreadcrumb />);
 
 storiesOf('AppHeader', module).add('default', () => <AppHeader />);
 
-storiesOf('Nav', module).add('default', () => <NavStory />);
+storiesOf('Nav', module).add('default', () => (
+  <NavStory goals={country.goals} goalInfos={sdgsMoreInfo.data} />
+));
 
 storiesOf('CountryHeader', module).add('default', () => {
   const { country_name } = country;
@@ -51,10 +57,12 @@ storiesOf('CountryHeader', module).add('default', () => {
 });
 
 storiesOf('CountrySelector', module).add('default', () => (
-  <CountrySelector
-    countries={countries}
-    onCountrySelect={e => console.log('Country selected: ', e)}
-  />
+  <div style={{ height: '100vh' }}>
+    <CountrySelector
+      countryList={countries}
+      onCountrySelect={e => console.log('Country selected: ', e)}
+    />
+  </div>
 ));
 
 storiesOf('GoalHeader', module).add('default', () => (
@@ -70,11 +78,7 @@ storiesOf('Series', module).add('default', () => (
 ));
 
 storiesOf('YesNoIndicator', module).add('default', () => (
-  <YesNoIndicator
-    goalInfo={goalInfo}
-    value={true}
-    description="The sky is blue."
-  />
+  <YesNoIndicator series={boolSeries} goalInfo={goalInfo} />
 ));
 
 storiesOf('MetricIndicator', module).add('default', () => (

